### DISCUSSION TITLE
[Support]: HA integration - long time between person count update and relative image sensor

### DISCUSSION THREAD

**Discussion by jiiins** [blakeblackshear/frigate#15169] [General Support]
### Describe the problem you are having

In HA, I need to take the snapshot of the image sensor (with detection boxes) whenever a camera's person count changes. However, in many cases the image sensor hasn't updated yet and shows the previous detection. I added a delay of 1 second before the snapshot but only helps some of the time. I've noticed that in some cases the image is updated several seconds later (up to 9).

Is there a faster way to pull the image with boxes from frigate directly, rather than relying on the image sensor?

Thanks

### Version

0.14.1-f4f3cfa

### Frigate config file

```yaml
.
```

### Relevant Frigate log output

```shell
.
```

### Relevant go2rtc log output

```shell
.
```

### FFprobe output from your camera

```shell
.
```

### Install method

Proxmox via Docker

### docker-compose file or Docker CLI command

```yaml
.
```

### Object Detector

Coral

### Network connection

Wired

### Camera make and model

.

**Comment by NickM-27** [comment]
yes, use the mqtt /events topic and pull the snapshot from the api

**Reply by jiiins** [reply]
I noticed that the snapshot changes from the initial image to include the object boxes. Is there a way to wait until the boxes are done?
